### PR TITLE
fix(ui5-color-picker): add tooltip to sliders

### PR DESCRIPTION
**Pull request by tsanislavgatev**
fixes: https://github.com/SAP/ui5-webcomponents/issues/10203

### CODE REVIEW

**Review by unazko — APPROVED** [review]
Tooltips are present and with accurate values.

File: packages/main/src/ColorPicker.ts [hunk 1]
```diff
@@ -366,41 +366,43 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 	}
 
 	_setMainColor(hueValue: number) {
-		if (hueValue <= 255) {
+		const hueValueMod = hueValue * 4.251;
```
Comment (vladitasev) [inline packages/main/src/ColorPicker.ts:369]: Usually, assigning to a new variable (`hueValueMod`) and using it throughout the function, is necessary when you need both the passed param (`hueValue`) and the modified one (`hueValueMod`) at the same time. But if you'll only use the modified one, the best would be to just have: `hueValue = hueValue * 4.251;` and continue using `hueValue` in the function as before.

Comment (vladitasev) [inline packages/main/src/ColorPicker.ts:369]: I know this magic number existed before, but this refactoring is an opportunity to set it to a constant.

File: packages/main/src/ColorPicker.ts [hunk 2]
```diff
@@ -436,7 +438,7 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 		// and HSL format, the color will be parsed to RGB
 		// 0 ≤ H < 360
 		// 4.251 because with 4.25 we get out of the colors range.
```
Comment (vladitasev) [inline packages/main/src/ColorPicker.ts:440]: This comment is now obsolete and should be deleted not to cause confusion.

**Pull request merged**